### PR TITLE
Handle spaces in parens in srcset descriptors. Fixes #180

### LINKED ISSUE #180: Should parse foo( ... ) with spaces as one descriptor
"Let descriptor list be the result of splitting unparsed descriptors on spaces."

I think this isn't quite what we want. Right now it only makes a difference if you do something like `foo( 2x )` which gets interpreted as 3 descriptors and `2x` is picked up.

### PR DESCRIPTION
**Pull request by zcorpan**

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -671,38 +676,130 @@ Parsing a <code>srcset</code> Attribute</h4>
...
+									If <var>current token</var> is not empty,
+									append <var>current token</var> to <var>descriptors</var>
+									and let <var>current token</var> be the empty string.
+									Set <var>state</var> to <i title>after token</i>.
+
+								<dt>U+002C COMMA (,)
+
+								<dd>
+									Advance <var>position</var> to the next character in <var>input</var>.
+									If <var>current token</var> is not empty,
```
Comment (yoavweiss) [inline index.bs:720]: or EOF?

File: index.bs [hunk 2]
```diff
@@ -671,38 +676,130 @@ Parsing a <code>srcset</code> Attribute</h4>
...
+									append <var>current token</var> to <var>descriptors</var>.
+									Jump to the next step in this algorithm.
```
Comment (yoavweiss) [inline index.bs]: It is not clear to me what the next step is... Maybe add a label to avoid confusion?

File: index.bs (hunk 1 above)
Comment (zcorpan) [inline index.bs:720]: EOF is handled lower down. It's the same except you don't advance to the next character since that makes little sense for EOF.

Comment (yoavweiss) [inline index.bs:720]: Wouldn't the above add the EOF to descriptors if the input ends with a comma?

Comment (yoavweiss) [inline index.bs:720]: Forget about it, read it all wrong

**Comment by yoavweiss** [comment]
Looks good!

**Pull request merged**